lamp/dashboards/dashboard.json

```diff
@@ -0,0 +1,373 @@
+            },
+            "linkedEntityGuids": null
+          },
+          {
+            "visualization": {
+              "id": "viz.line"
+            },
+            "layout": {
+              "column": 3,
+              "row": 4,
+              "height": 3,
+              "width": 4
+            },
+            "title": "Requests per second",
+            "rawConfiguration": {
+              "legend": {
+                "enabled": true
+              },
+              "nrqlQueries": [
+                {
+                  "accountId": 0,
+                  "query": "SELECT average(`net.requestsPerSecond`) * uniqueCount(entityName) as 'Requests' FROM ApacheSample TIMESERIES until 1 minute ago"
+                }
+              ],
+              "yAxisLeft": {
+                "zero": true
+              }
+            },
+            "linkedEntityGuids": null
+          },
+          {
+            "visualization": {
+              "id": "viz.billboard"
+            },
+            "layout": {
+              "column": 7,
+              "row": 4,
+              "height": 3,
+              "width": 5
+            },
+            "title": "Worker status",
+            "rawConfiguration": {
+              "dataFormatters": [],
+              "nrqlQueries": [
+                {
+                  "accountId": 0,
+                  "query": "SELECT average(`server.idleWorkers`) * uniqueCount(entityName) as 'Idle Workers', average(`server.busyWorkers`) * uniqueCount(entityName) as 'Busy Workers', average(`server.scoreboard.totalWorkers`) * uniqueCount(entityName) as 'Total Workers', average(`server.scoreboard.readingWorkers`) * uniqueCount(entityName) as 'Reading Request', average(`server.scoreboard.writingWorkers`) * uniqueCount(entityName) as 'Writing', average(`server.scoreboard.loggingWorkers`) * uniqueCount(entityName) as 'Logging', average(`server.scoreboard.finishingWorkers`) * uniqueCount(entityName) as 'Finishing', average(`server.scoreboard.closingWorkers`) * uniqueCount(entityName) as 'Closing Connection', average(`server.scoreboard.keepAliveWorkers`) * uniqueCount(entityName) as 'Keep Alive', average(`server.scoreboard.dnsLookupWorkers`) * uniqueCount(entityName) as 'DNS Lookup', average(`server.scoreboard.idleCleanupWorkers`) * uniqueCount(entityName) as 'Idle Cleanup', average(`server.scoreboard.startingWorkers`) * uniqueCount(entityName) as 'Starting' FROM ApacheSample "
+                }
+              ],
+              "thresholds": []
+            },
+            "linkedEntityGuids": null
+          },
+          {
+            "visualization": {
+              "id": "viz.markdown"
+            },
+            "layout": {
+              "column": 1,
+              "row": 7,
+              "height": 3,
+              "width": 2
+            },
+            "title": "",
+            "rawConfiguration": {
+              "text": "# MySQL\n![MySQL Logo](https://newrelic.com/sites/default/files/2020-10/mysql.svg)"
```

Comment:
I'm not sure if we want to reference pack images from external sources, wouldn't we want these to be part of the pack meta data and reside within the repo so we can call these images from the API?

Reply:
I added them directly to the repo